tools/clang/test/HLSL/cpp-errors-hv2015.hlsl

```diff
@@ -576,8 +571,8 @@ void expressions()
   internal->fn();                 // expected-error {{operator is not supported}}
   local_i = (int3) { 1, 2, 3 };   // expected-error {{compound literal is unsupported in HLSL}}
 
-  // `class` ok, but component count should be checked earlier (1 to 4 uniform components):
-  Texture2D<::c_outer_fn> local_texture;
+  // ok, but component count should be checked earlier (1 to 4 uniform components):
+  Texture2D<::c_outer_td> local_texture;
```

Comment:
I feel like the comment is out of place now.  Before, it was there because the class used had no members (was empty), and it was a comment that there was no Sema error checking this (DXIL backend checked in CGHLSLMS).  Now that it's just a normal float4, what does the comment mean to a future reader of this line?

Maybe we should remove the comment now?